src/utils.js

```diff
@@ -81,7 +81,7 @@ export function getTableWidgetAncestor( selection ) {
 export function getSelectedTableCells( selection ) {
 	const cells = [];
 
-	for ( const range of selection.getRanges() ) {
+	for ( const range of sortRanges( selection.getRanges() ) ) {
```

Comment:
There are 2 more places that would benefit from a public helper:
1. https://github.com/ckeditor/ckeditor5-restricted-editing/blob/master/src/restrictededitingmodenavigationcommand.js#L110-L116
2. https://github.com/ckeditor/ckeditor5-engine/blob/master/src/model/range.js#L933-L935

WDYT?

Reply:
Hard to say as sorting ranges can be done in more ways (not only by using the start position).

I'll report a followup though as I realised that e.g. the `isEqual()` check in my code is not necessary.

Reply:
Yeah, `isBefore/After` does the trick.

Reply:
ckeditor/ckeditor5#6414 done